lib/fs/basicfs_xattr_bsdish.go

```diff
@@ -0,0 +1,93 @@
+// Copyright (C) 2022 The Syncthing Authors.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//go:build freebsd || netbsd
+// +build freebsd netbsd
+
+package fs
+
+import (
+	"errors"
+	"fmt"
+	"unsafe"
+
+	"golang.org/x/sys/unix"
+)
+
+func listXattr(path string) ([]string, error) {
+	var attrs []string
+
+	// List the two namespaces explicitly and prefix any results with the
+	// namespace name.
+	namespacePrefixes := [...]string{unix.EXTATTR_NAMESPACE_USER: "user.", unix.EXTATTR_NAMESPACE_SYSTEM: "system."}
```

Comment:
What is this syntax? I only know `[...]` for arrays as below, but this is a int to string map.

Also make both a const outside of func?

Reply:
> but this is a int to string map.

Haha! So you might think, but then I fooled you. This is in fact an array constant with values of the form `offset: value`.

But yeah, this could be moved outside.